src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BindingBuildTest.cs

```diff
@@ -534,6 +534,8 @@ public void AppWithSingleJar ()
 
 			var builder = CreateApkBuilder ();
 			Assert.IsTrue (builder.Build (proj), "first build should succeed");
+			Assert.IsTrue (builder.DesignTimeBuild (proj), "Design time build should succeed.");
+			Assert.IsFalse (builder.Output.IsTargetSkipped ("AddBindingsToCompile"), "AddBindingsToCompile should run.");
```

Comment:
I'm confused a little by this assertion. If we do a design-time build *after* a build, why does `AddBindingsToCompile` need to run?

Reply:
It adds the files generated for `AndroidJavaSource` bindings to the `Compile` list. 
If this target is skipped in a DTB, we loose intelisense.